docs/proposals/work-pause/README.md

```diff
@@ -0,0 +1,625 @@
+	// set to false indicates the pause is canceled.
+	// Default value is false.
+	// Note: The extension design is not a focal point of the current proposal.
+	ResourceMatchPause *bool `json:"ResourceMatchPause,omitempty"`
+
+	// SchedulePause specifies the pause scheduling of resources matched by the policy,
+	// set to false indicates the pause is canceled.
+	// Default value is false.
+	// Note: The extension design is not a focal point of the current proposal.
+	SchedulePause *bool `json:"schedulePause,omitempty"`
+
+	// DistributionPauseStrategy specifies the strategy for pausing resource distribution.
+	DistributionPauseStrategy *DistributionPauseStrategy `json:"distributionPauseStrategy,omitempty"`
+}
+
+type DistributionPauseStrategy struct {
+	// PauseAllClusters specifies the overall suspension of resource distribution across all clusters.
+	// If PauseAllClusters field is set, PauseClusterAffinity field will be ignored.
+	// Default value is false.
+	PauseAllClusters bool `json:"pauseAllClusters,omitempty"`
+
+	// PauseClusterAffinity specifies the cluster list where resource distribution needs to be paused.
+	PauseClusterAffinity *ClusterAffinity `json:"pauseClusterAffinity,omitempty"`
+}
+```
+
+<!--
+对于资源匹配暂停，是指当用户创建 PropagationPolicy/ClusterPropagationPolicy 之后，Karmada 系统不进行资源匹配进而生成 ResourceBinding/ClusterResourceBinding 对象。
+
+对于调度暂停，是指在 Karmada 系统内，ResourceBinding/ClusterResourceBinding 资源创建或更新后，Karmada 调度器不计算调度结果，直至调度恢复。
+-->
+
+For resource matching suspension `ResourceMatchPause`, it refers to the situation where, after the user creates a `PropagationPolicy/ClusterPropagationPolicy`, the Karmada system does not proceed with resource matching to generate `ResourceBinding/ClusterResourceBinding` objects.
+
+For scheduling suspension `SchedulePause`, it refers to the situation within the Karmada system where, after the creation or update of `ResourceBinding/ClusterResourceBinding` resources, the Karmada scheduler does not calculate the scheduling results until the scheduling is resumed.
+
+##### ResourceBinding/ClusterResourceBinding
+
+```go
+// ResourceBindingSpec represents the expectation of ResourceBinding.
+type ResourceBindingSpec struct {
+    ...
+
+	PauseStrategy *PauseStrategy `json:"pauseStrategy,omitempty"`
+}
+
+type PauseStrategy struct {
+	// SchedulePause specifies the pause scheduling of resources matched by the policy,
+	// set to false indicates the pause is canceled.
+	// Default value is false.
+	// Note: The extension design is not a focal point of the current proposal.
+	SchedulePause *bool `json:"schedulePause,omitempty"`
+
+	// DistributionPauseStrategy specifies the strategy for pausing resource distribution.
+	DistributionPauseStrategy *DistributionPauseStrategy `json:"distributionPauseStrategy,omitempty"`
+}
+```
+
+A new ConditionType called `SchedulePause` is added to describe the scheduling pause status of the ResourceBinding/ClusterResourceBinding resources.
```

Comment:
I think we should also generate some events similar to `SchedulePause`, right?

Reply:
Thanks, I think yes!